paymentsheet/src/main/java/com/stripe/android/paymentsheet/customersheet/CustomerSheetResult.kt

```diff
@@ -45,7 +68,8 @@ sealed class CustomerSheetResult {
  */
 @ExperimentalCustomerSheetApi
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Parcelize
```

Comment:
I think we should try to rework this in such a way that doesn't require parcelable stuff.